src/synthesizer/emission_models/base_model.py

```diff
@@ -414,10 +421,12 @@ def _init_operations(
                 luminosity when computing dust emission.
             fesc (float):
                 The escape fraction.
+            lam_mask (ndarray):
+                The mask to apply to the wavelength array.
         """
         # Which operation are we doing?
         if self._is_extracting:
-            Extraction.__init__(self, grid, extract, fesc)
+            Extraction.__init__(self, grid, extract, fesc, lam_mask)
```

Comment:
We probably want to propagate the lam mask to all 4 operations to give full flexibility (e.g. applying attenuation in specific ranges). Probably niche but it's maybe a bit confusing conceptually that you can pass a lam_mask into any model but it only gets used from an extraction.

Reply:
Kind of, but if you're masking by wavelength you probably want to do enforce that on an extraction or generation step anyway. Doing it on an attenuation seems a bit strange

Reply:
Yea, I resolved this because I came to the conclusion if it ever is needed for some reason it'll be trivial to add then.